java-api-core/src/com/kinvey/java/model/KinveyUpdateSingleItemError.java

```diff
@@ -0,0 +1,45 @@
+package com.kinvey.java.model;
+
+import com.google.api.client.json.GenericJson;
+import com.google.api.client.util.Key;
+
+import java.io.IOException;
+
+public class KinveyUpdateSingleItemError extends IOException {
+
+    public KinveyUpdateSingleItemError(Exception e, GenericJson entity) {
+        this.entity = entity;
+        this.errmsg = e.getMessage();
+    }
+
+    @Key
+    private GenericJson entity;
+    @Key
+    private long code;
+    @Key
+    private String errmsg;
+
+    public String getErrmsg() {
```

Comment:
since this is already an Exception/Error class, I think it's fine to give this a better name like `message` instead of `errmsg`. What do you think?

Reply:
Fixed.

Reply:
I see you fixed the other issues, but this one still as it was. maybe you forgot to commit this file